src/NexusMods.App.UI/WorkspaceSystem/NewTabPage/NewTabPageViewModel.cs

```diff
@@ -34,7 +34,10 @@ public NewTabPageViewModel(PageDiscoveryDetails[] discoveryDetails)
             _itemSource
                 .Connect()
                 .MergeMany(item => item.SelectItemCommand)
-                .InvokeCommand(ChangePageCommand)
+                .SubscribeWithErrorLogging(pageData =>
+                {
+                    WorkspaceController.OpenPage(pageData, new OpenPageBehavior(new OpenPageBehavior.ReplaceTab(PanelId, TabId)));
```

Comment:
`OpenPageBehavior.ReplaceTab` is the `ReplaceTab` record, so there can't be a static function with the same name.

Reply:
Rename the records to `ReplaceTabData` or something along those lines?

Reply:
I don't see the benefit of this. This would essentially require duplicating the record with its docs to create a new function that just calls the constructor. You'd have to keep both the record and function and their docs up-to-date.

Reply:
The benefit would be that people that need to pass a OpenPageBehaviour don't need to go through the awkward: 
```csharp 
new OpenPageBehavior(new OpenPageBehavior.ReplaceTab(PanelId, TabId))
```
and could instead do:
```csharp
OpenPageBehavior.ReplaceTab(PanelId, TabId)
```

Reply:
I'd rather have the awkward constructor and one exact way of using this, than an awkward constructor and a helper function that calls the constructor, which has to be updated with the constructor.

Reply:
Very well, maybe correct solution would instead have been something with interfaces and inheritance or something, but it's not worth wasting any more time on this.